src/eth/utils.ts

```diff
@@ -1,11 +1,12 @@
+import type { KeyringAccount } from '../api';
 import { EthAccountType } from '../api';
-import type { InternalAccountType } from '../internal';
 
 /**
  * Checks if the given type is an EVM account type.
+ *
  * @param type - The type to check.
  * @returns Returns true if the type is an EVM account type, false otherwise.
  */
-export function isEvmAccountType(type: InternalAccountType | string): boolean {
+export function isEvmAccountType(type: KeyringAccount['type']): boolean {
```

Comment:
We have to keep the `| string` here. @montelaidev did add this last time cause in some context we have a account's type as a `string` and we wanna check if this is actually an EVM account or not.

We initially kept the `InternalAccountType` on purpose to give an hint of the "known" (supported) account types on the keyring API.